app/lua53/luaconf.h

```diff
@@ -401,7 +401,7 @@
 @@ l_sprintf is equivalent to 'snprintf' or 'sprintf' in C89.
 ** (All uses in Lua have only one format item.)
 */
-#if !defined(LUA_USE_C89) && !defined(LUA_USE_ESP8266)
+#if !defined(LUA_USE_C89) || defined(LUA_USE_ESP8266)
```

Comment:
Is `LUA_USE_C89` not defined when we're building for the ESP8266?

Reply:
I'll fix that.